lib/menu.rb

```diff
@@ -0,0 +1,14 @@
+class Menu
+
+  attr_reader :content
+
+  def initialize
+    @content = {"Fish and chips" => 7, "Steak and chips" => 10,
+                "Shepherd's Pie" => 5, "Bangers and mash" => 5}
+  end
+
+  def read
+    @content.each {|dish, price| puts dish.ljust(20) + price.to_s.rjust(2)}
```

Comment:
Space between { and | missing.<br>Space missing inside }.<br>Do not write to stdout. Use Rails' logger if you want to log.